src/module.js

```diff
@@ -117,7 +117,7 @@ _.extend(Marionette.Module.prototype, Backbone.Events, {
       Marionette,
       Backbone.$, _,
       customArgs
-    ]);
+    ], true);
```

Comment:
`lodash@3` has the opposite behaviour of `underscore` and `lodash@2` for this flag